src/features/autosuggest/autosuggest.ts

```diff
@@ -7,6 +7,16 @@ import type { SuggestFixedOptions } from './suggest-fixed-options.type';
 import type { SuggestRequestParameters } from './suggest-request.type';
 import type { SuggestResponse } from './suggest-response.type';
 
+const FIXED_OPTIONS: SuggestFixedOptions = {
+  request_type: 'suggest',
+};
+
+export function isAutosuggestOptions(options: Record<string, any>): options is AutosuggestOptions {
+  return Object.entries(FIXED_OPTIONS).every(([key, value]) => {
+    return options[key] !== undefined && options[key] === value;
```

Comment:
you can probably skip the `options[key] !== undefined` part

Reply:
Yes we can!